config/features.yaml

```diff
@@ -75,3 +75,14 @@ github_accounts:
         aggregate: True
         tags: ['Github Account']
         emojis: ['ID_BUTTON']
+
+# Linkedin account extraction from profile edit url
+linkedin_accounts:
+        query_string: 'source_short:"WEBHIST" AND
+          url:"https://www.linkedin.com/in/" AND "/edit/"'
```

Comment:
Limit the "/edit/" search to url field:
url:"https://www.linkedin.com/in/" AND url:"/edit/"'

Reply:
overlooked that. done!